wger/manager/templates/workout/view.html

```diff
@@ -144,6 +144,49 @@ <h4 class="modal-title">{% trans "Export calendar file" %}</h4>
     </div>
 </div>
 
+<div class="modal fade" id="export-workout-popup">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
+                <h4 class="modal-title">{% trans "Export workout data" %}</h4>
+            </div>
+            <div class="modal-body">
+                <form class="form-horizontal">
+                    <div class="form-group">
+                        <div>
+                            <label for="id_format" class="col-md-3 control-label">
+                                {% trans "Format: " %}
+                            </label>
+                        </div>
+                        <div class="col-md-9 row" style="padding-left: 35px;">
```

Comment:
Maybe you should have added styling in CSS file

Reply:
Noted. I've done this.